extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -120,11 +144,29 @@ public function isValidHeroDataItem( $item, $heroData ) {
 	 * required for testing mobile template rendering
 	 * @return bool
 	 */
-	public function isWikiaMobile() {
+	public function isMobile() {
 		return \F::app()->checkSkin( 'wikiamobile' );
 	}
 
 	/**
+	 * @return bool
+	 */
+	public function isMercury() {
```

Comment:
Yes, and I actually think this is the type of method that does not really fit this service - it's something worth moving to a separate piece of code